src/LogTarget.php

```diff
@@ -41,9 +41,27 @@ public function __construct($module, $config = [])
         $this->tag = uniqid();
     }
 
+    /**
+     * Processes the given log messages.
+     * This method will filter the given messages with [[levels]] and [[categories]].
+     * And if requested, it will also export the filtering result to specific medium (e.g. email).
+     * @param array $messages log messages to be processed. See [[\yii\log\Logger::messages]] for the structure
+     * of each message.
+     * @param bool $final whether this method is called at the end of the current application
+     * @throws \yii\base\Exception
+     */
+    public function collect($messages, $final)
```

Comment:
Why moving the method?

Reply:
Same reason as above (reformatting code). I think in coding standards is defined to list `public` methods above `protected` ones.